modules/relax/layouts/relax.cfm

```diff
@@ -16,8 +16,6 @@
     <title>ColdBox Relax - RESTful Tools For Lazy Experts</title>
     <!--- favicon --->
     <link href="#prc.root#/includes/images/favicon.ico" rel="shortcut icon" type="image/x-icon" />
-    <!--- SES --->
```

Comment:
I suppose all affected links could be changed from:
<link href="#prc.root#/includes/images/favicon.ico" rel="shortcut icon" type="image/x-icon" />
to
<link href="/#prc.root#/includes/images/favicon.ico" rel="shortcut icon" type="image/x-icon" />

I can help. Is it a matter of going through and doing something like that throughout?